src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/settings/ManagedIndexSettings.kt

```diff
@@ -39,7 +39,7 @@ class ManagedIndexSettings {
         val POLICY_ID: Setting<String> = Setting.simpleString(
             "index.opendistro.index_state_management.policy_id",
             Setting.Property.IndexScope,
-            Setting.Property.Dynamic
+            Setting.Property.Deprecated
```

Comment:
Were you able to verify this doesn't cause any issues besides just deprecation warning?

Reply:
I tried create index with policyID setting, seeing this message:

#! Deprecation: [index.opendistro.index_state_management.policy_id] setting was deprecated in Elasticsearch and will be removed in a future release! See the breaking changes documentation for the next major version.

not sure if we want this. I prefer not set it to Deprecated. Just inform user in the documentation

Reply:
Got it, feel free to remove, the message about breaking changes major version is unneeded.

Reply:
still using Dynamic